app/styleguide/routes/layout.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div :class="b()">
+    <c-header/>
+    <c-content/>
+    <c-footer/>
+  </div>
+</template>
+
+<script>
+  import sNavigation from '@/styleguide/components/s-navigation';
+  import cHeader from '@/components/c-header';
+  import cContent from '@/components/c-content';
+  import cFooter from '@/components/c-footer';
+
+  export default {
+    name: 'layout',
+    components: {
+      sNavigation,
+      cHeader,
+      cContent,
+      cFooter
+    },
+    // mixins: [],
+
+    // props: {},
+    // data() {
+    //   return {};
+    // },
+
+    // computed: {},
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    // mounted() {},
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    // methods: {},
+    // render() {},
+  };
+</script>
+
+<style lang="scss">
+
+  body {
+    display: block;
```

Comment:
- [ ] It would also be ok to wrap the `s-navigation` inside `l-default`.

Reply:
Let me know if my frame solution doesn't make sense :-)